frontend/packages/client/src/components/Community/CommunityEditorDetails/AddressForm.js

```diff
@@ -0,0 +1,107 @@
+import React, { useCallback, useMemo } from 'react';
+import { AddButton, Loader, WrapperResponsive } from 'components';
+import FormFields from './FormFields';
+
+export default function AddressForm({
+  title,
+  description,
+  loadingUsers = false,
+  addrList,
+  listName,
+  errors,
+  onDeleteAddress,
+  onAddAddress,
+  addrType = 'Admins',
+  register,
+  update,
+  isValid,
+  submitComponent,
+  removeInnerForm,
+  isSubmitting = false,
+  handleSubmit,
+  showValidIcon,
```

Comment:
I think this ended up having too many props. Another option to have less props passed could be pass the `formFieldsComponent` as a component, then the `AddressForm` component doesn't need to get that many props. The downside of it is that the parent component is aware of the composition (here https://github.com/DapperCollectives/CAST/pull/337/files#diff-bd4d5820dc252034709a92f093bb51951a9afe419194b621250729dcf34cbba2R102 we would need to pass down the `formFieldsComponent` )

Reply:
I think it is okay for now. Another options could be to pass a props object and use a helper to construct the props. I have done that in the past and it can be a good way to manage a bunch of props. something like `const addressProps =  getAddressProps(arg1, arg2, arg3)`